src/mixins/filing-template-mixin.ts

```diff
@@ -868,7 +872,8 @@ export default class FilingTemplateMixin extends DateMixin {
       }
     ))
 
-    if (filing.restoration.relationships) {
+    // store relationships
+    if (filing.restoration.relationships?.length > 0) {
       this.setRestorationRelationships(filing.restoration.relationships)
     }
 
```

Comment:
I might be able to fix another schema issue in this file as part of this PR (if I find it soon).

ATM I am unable to file my test restoration extension.